index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Gentium+Plus:wght@700&family=Poppins&family=Roboto:wght@700&display=swap"
+    rel="stylesheet">
+  <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+  <header>
+    <div class="logo">Bright Ezeamaka</div>
+    <div class="dropdown">
+      <img src="./Images/Icon - Menu.png" alt="">
+    </div>
+  </header>
+  <section class="headline">
+    <div class="headline-container">
+      <h1>Hey There. I'm Bright</h1>
+      <p>
+        <span>I’am a Software Developer</span>
+      </p>
+      <p>
+        I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded,
+        don’t hestiate to contact me.
+      </p>
+      <ul>
+        <li class="icon"><i class="fa-brands fa-github"></i></li>
+        <li><i class="fa-brands fa-linkedin"></i></li>
+        <li><i class="fa-brands fa-angellist"></i></li>
+        <li><i class="fa-brands fa-twitter"></i></li>
+        <li><i class="fa-solid fa-m"></i></li>
+
+      </ul>
```

Comment:
- [ ] You did a good job wrapping your `li` elements with `ul`, but I think it would be better if you make sure that each icon has an `a` tag because apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href` attribute.